File: PROGRAMMERS/LEVEL1/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;

    int num_1 = 0;
    int num_2 = 0;
    for (int i = 0; i < s.length(); i++) {
        if (s[i] == 'p' || s[i] == 'P') num_1++;
        if (s[i] == 'y' || s[i] == 'Y') num_2++;
    }
    if (num_1 != num_2) answer = false;

    return answer;
}

File: PROGRAMMERS/LEVEL1/문자열 내 p와 y의 개수.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.*;

class Solution {
    boolean solution(String s) {
        boolean answer = true;

        int num_1 = 0;
        int num_2 = 0;
        for (int i = 0; i < s.length(); i++) {
        	if (s.charAt(i) == 'p' || s.charAt(i) == 'P') num_1++;
        	if (s.charAt(i) == 'y' || s.charAt(i) == 'Y') num_2++;
        }
        if (num_1 != num_2) answer = false;

        return answer;
    }
}